build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,9 +7,10 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.0.2";
-        public static readonly string JitVersion = "1.0.2";
-        public static readonly string CoreCLRVersion_Latest = "1.1.0-beta-24415-01";
-        public static readonly string JitVersion_Latest = "1.1.0-beta-24415-01";
+        // TODO: Can these versions be updated by automation?
```

Comment:
Yes, Maestro already knows how to update the CoreCLRVersion and JitVersion properties:

https://github.com/dotnet/core-setup/blob/master/build_projects/update-dependencies/UpdateFilesTargets.cs#L200-L215

I'm not sure why we added these "Latest" properties, or why they are needed now that the property values are both insync.  Can we remove the "_Latest" properties?  @ianhays